### DISCUSSION TITLE
New to Github

### DISCUSSION THREAD

**Discussion by kendaWassel** [community/community#77041] [New to GitHub]
hello everyone,
I am new to github. I’ve been following tutoials etc on how to set up pages. Now, i’m trying to move my codepen HTML and CSS over here. HTML is working, but CSS is not.
can you help me please.
https://kendawassel.github.io/Epic_Design/

**Comment by guillemalqueza** [comment]
Try removing the / before the css folder where you are defining the href.

```
    <link rel="stylesheet" href="css/all.min.css">
    <link rel="stylesheet" href="css/normalize.css">
    <link rel="stylesheet" href="css/dash.css">
```
